microsoft-edge/web-platform/tracking-prevention.md

```diff
@@ -146,7 +146,12 @@ The following table shows the enforcement actions and mitigations that are appli
 
 ### InPrivate behavior
 
-In Microsoft Edge 79, the default behavior was to apply Strict mode protections in InPrivate.  In Microsoft Edge 80, this behavior was replaced by a switch in `edge://settings/privacy` that allows users to decide whether to apply Strict mode protections or to keep their regular settings while browsing InPrivate.
+By default, InPrivate windows in Microsoft Edge use the same tracking prevention settings as regular windows.
+
+To always apply the Strict level of tracking prevention to InPrivate windows:

```

Comment:
```suggestion
To always apply the "Strict" level of tracking prevention to InPrivate windows:
```
backticks or quotes like a few lines below could be used in addition to init cap, to differentiate vs. regular words